checkstyle/suppressions.xml

```diff
@@ -43,7 +43,7 @@
 
     <suppress
             checks="(ParameterNumber)"
-            files="(ClusterData).java"
+            files="(ClusterData|TopicData).java"
```

Comment:
I don't add any suppressions here in this file. Instead we add them (if necessary) in the place we want to supress, using comments:

```
// CHECKSTYLE:OFF:ParameterNumber
// My code that needs supression.
// CHECKSTYLE:ON:ParameterNumber
```